README.md

```diff
@@ -1 +1,45 @@
-# Hello-World
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Hello World README file</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+**[Hello WOrld]** is a simple exercise of the concepts we have learned so far. Concepts such as git, GitHub, GitHub flow and other. Just think of it as a practice run for me, as a new micronaut.
+
+## 🛠 Built With <a name="Git"></a>
+<p>I made use of Git and GitHub.</p>
+<p>I also made use of simple html and CSS, as required of me</p>
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[What do you think about my project?]**
+
+  - [Answer_1]
+
+- **[Is it really confusing on the first day? Will I improve?]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.